exercises/chapter3/src/Data/AddressBook.purs

```diff
@@ -6,17 +6,21 @@ import Control.Plus (empty)
 import Data.List (List(..), filter, head)
 import Data.Maybe (Maybe)
 
+-- ANCHOR: address
 type Address =
   { street :: String
   , city   :: String
   , state  :: String
   }
+-- ANCHOR_END: address
```

Comment:
Anchors are the most robust way to link to the source, but may become annoying when reading the source.